Dockerfile

```diff
@@ -15,7 +15,7 @@ COPY . /opt/sworchestrator/
 WORKDIR $python_lib/iotp-orchestrator
 
 RUN \
-    adduser --comment "${ORCHESTRATOR_USER}" ${ORCHESTRATOR_USER} && \
+    adduser --comment "${ORCHESTRATOR_USER}" -u 1500 ${ORCHESTRATOR_USER} && \
```

Comment:
1500 sounds like a "magic number"... which user groups is 1500? Is somekind of "standardized" user id in docker?

(Just curiosity)

Reply:
Yeah, that's correct. It's a "magic_ number". We need to specify an explicit UID for the orchestrator user, actually It could be any number from 1000, but I wasn't sure if the image or any process creates a user with UID 1000, 1001... 

So I decided to use 1500.